src/urdf_parser_py/urdf.py

```diff
@@ -352,18 +352,19 @@ def joint_type(self, value): self.type = value
 class Link(xmlr.Object):
     def __init__(self, name=None, visual=None, inertial=None, collision=None,
                  origin=None):
+        self.aggregate_init()
         self.name = name
-        self.visual = visual
-        self.inertial = inertial
-        self.collision = collision
+        self.visuals = []
+        self.inertials = inertial
```

Comment:
Since a link may only have one `<interial>` tag I think this should stay as `intertials`

Reply:
You mean inertial correct?